src/codegen/cpu.rs

```diff
@@ -129,7 +130,7 @@ impl<'a, Tgt: Target<Level = CpuMemoryLevel>> CpuCodeGenerator<'a, Tgt> {
         for (idx, input_tensor) in top_arg_tensors.iter().enumerate() {
             // Open and mmap the data.
             let value_cnt = input_tensor.0.volume();
-            let byte_cnt = value_cnt * DimSize::from(input_tensor.spec().dtype().size());
+            let byte_cnt = value_cnt.get() * u32::from(input_tensor.spec().dtype().size());
```

Comment:
> Let's not use nightly Rust.

Sure.

> In any case, this is a complicated solution to a non-problem. `u32` is fine or, alternately, both value_cnt and `dtype().size()` are themselves non-zero. I don't see a performance benefit to using a nz type here, so I would go with whichever is more concise.

My comment was insufficient. I meant by "information loss" the change in terms of the code to read. When we read the updated code, it is kind of hard to know that the type (`u32`) is associated with `DimSize`. That said, I now feel it's not a big deal since Rust does not allow implicit conversions... Sorry for the confusion!